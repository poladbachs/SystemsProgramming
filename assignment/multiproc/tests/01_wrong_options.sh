"$1" --not-existing-arg >/dev/null
echo "Exit code is: $?"
