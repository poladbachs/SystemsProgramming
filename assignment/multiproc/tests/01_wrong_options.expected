Usage: ./multiproc [options] [file]...

Arguments:
  [file]...       The list of regular files on which the program should
                  apply the processing and summarize operations. The
                  list can also contain some directories. In case of a
                  directory, the program will apply the processing and
                  summarizing operation on all regular files within the
                  directory and its sub-directories.

Options:
  -p <prog>, --proc <prog>             The program for the processing
                                       operation (default: cat)
  -s <prog>, --summary <prog>          The program for the summarize
                                       operation (default: cat)
  -w <workers>, --workers <workers>    The number of forked processes
                                       (default: 5, max: 20)
  -h, --help                           Display this help and exit
Exit code is: 1
